packs/data/fall-of-plaguestone.db/spear-launcher.json

```diff
@@ -0,0 +1,165 @@
+{
+    "_id": "jpw5eHse4EcmYSp6",
+    "data": {
+        "attributes": {
+            "ac": {
+                "value": 18
+            },
+            "hardness": 8,
+            "hp": {
+                "details": "",
+                "max": 32,
+                "temp": 0,
+                "value": 32
+            },
+            "stealth": {
+                "details": "<p>(trained)</p>",
+                "value": 10
+            }
+        },
+        "creatureType": "",
+        "customModifiers": {},
+        "details": {
+            "description": "<p>An old heavy crossbow is hidden in a pile of trash, loaded with a wooden spear, and connected to the rope holding the door.</p>",
+            "disable": "<p>@Check[type:thievery|dc:18|traits:secret]{Thievery (trained)} on the rope allows a PC to tie the rope off and open the door without setting off the trap.</p>",
```

Comment:
```suggestion
            "disable": "<p>@Check[type:thievery|dc:18|traits:action:disable-device]{Thievery (trained)} on the rope allows a PC to tie the rope off and open the door without setting off the trap.</p>",
```

Reply:
Oh, snap. I didn't even realize that was a thing; I didn't see it in the Rollable DC Button thing.